frontend/src/bundles/common/components/icon/enums/icon-components.enum.ts

```diff
@@ -1,6 +1,8 @@
 import ArrowDownIcon from '~/assets/img/icons/arrow-down-icon.svg?react';
 import LogoIcon from '~/assets/img/icons/logo-icon.svg?react';
+import MoonIcon from '~/assets/img/icons/moon-icon.svg?react';
 import NotFoundIcon from '~/assets/img/icons/not-found-icon.svg?react';
+import SunIcon from '~/assets/img/icons/sun-icon.svg?react';
```

Comment:
Are these icons not available in Heroicons? If they are, it's better to use them from the library.

Reply:
Well, yes i extracted the svg from the page from hero icons, but since we are using https://www.npmjs.com/package/vite-plugin-svgr I stuck with that. 

Of course I could change it at request, what do you think?